compiler/packages/babel-plugin-react-compiler/src/ReactiveScopes/MergeReactiveScopesThatInvalidateTogether.ts

```diff
@@ -78,6 +88,8 @@ import {
 export function mergeReactiveScopesThatInvalidateTogether(
   fn: ReactiveFunction
 ): void {
+  log("MergeReactiveScopesThatInvalidateTogether");
+  log(printReactiveFunction(fn));
```

Comment:
Makes sense if we want to keep this for debugging. 

Maybe we should consider passing in a lambda that wraps `printReactiveFunction` or running a rollup transform that strips `log`, `logger.logDiagnostic`, etc argument expressions so that prod build systems don't need to call these functions.

Doesn't have to be this PR, just wanted to record for the future..